appshell/cefclient_mac.mm

```diff
@@ -334,8 +334,15 @@ - (void)windowWillEnterFullScreen:(NSNotification *)notification {
     }
     if (customTitlebar) {
         [customTitlebar setHidden:YES];
+        NSWindow *window = [notification object];
+
+        // Since we have nuked the title, we will have
+        // to set the string back as we are hiding the
+        // custom title bar.
+        [window setTitle:[customTitlebar titleString]];
```

Comment:
@nethip I did run into a console message viz., the following. Was that happening earlier also?

2016-05-12 22:52:11.306 Brackets[2370:84493] NSWindow warning: adding an unknown subview: <CustomTitlebarView: 0x608000131440>. Break on NSLog to debug.
2016-05-12 22:52:11.351 Brackets[2370:84493] Call stack:
(
	0   AppKit                              0x00007fff851c88bb -[NSThemeFrame addSubview:] + 107
	1   AppKit                              0x00007fff851c8600 -[NSView addSubview:positioned:relativeTo:] + 211
	2   Brackets                            0x00000001000588bb -[ClientWindowDelegate windowDidBecomeKey:] + 603
	3   CoreFoundation                      0x00007fff9768ebbc __CFNOTIFICATIONCENTER_IS_CALLING_OUT_TO_AN_OBSERVER__ + 12
	4   CoreFoundation                      0x00007fff9768eb4f ___CFXRegistrationPost_block_invoke + 63
	5   CoreFoundation                      0x00007fff9768eac7 _CFXRegistrationPost + 407
	6   CoreFoundation                      0x00007fff9768e832 ___CFXNotificationPost_block_invoke + 50
	7   CoreFoundation                      0x00007fff9764b5e2 -[_CFXNotificationRegistrar find:object:observer:enumerator:] + 1922
	8   CoreFoundation                      0x00007fff9764a835 _CFXNotificationPost + 693
	9   Foundation                          0x00007fff88399fda -[NSNotificationCenter postNotificationName:object:userInfo:] + 66
	10  AppKit                              0x00007fff852e6048 -[NSWindow becomeKeyWindow] + 1425
	11  AppKit                              0x00007fff852e5a75 _NXSendWindowNotification + 252
	12  AppKit                              0x00007fff852e5378 -[NSWindow _changeKeyAndMainLimitedOK:] + 868
	13  AppKit                              0x00007fff853af173 -[NSWindow _makeKeyRegardlessOfVisibility] + 98
	14  AppKit                              0x00007fff852e8429 NSPerformVisuallyAtomicChange + 147
	15  AppKit                              0x00007fff853af0af -[NSWindow makeKeyAndOrderFront:] + 79
	16  Brackets                            0x0000000100059a5f -[ClientAppDelegate createApp:] + 2575
	17  Foundation                          0x00007fff883c7f4e -[NSObject(NSThreadPerformAdditions) performSelector:onThread:withObject:waitUntilDone:modes:] + 1115
	18  Foundation                          0x00007fff883c7a75 -[NSObject(NSThreadPerformAdditions) performSelectorOnMainThread:withObject:waitUntilDone:] + 131
	19  Brackets                            0x000000010005b276 main + 3030
	20  Brackets                            0x0000000100003d54 start + 52
	21  ???                                 0x0000000000000003 0x0 + 3
)
Got xpc error message: Connection interrupted
2016-05-12 22:54:16.043 Brackets[2370:85805] Communications error: <OS_xpc_error: <error: 0x7fff76257b90> { count = 1, contents =
	"XPCErrorDescription" => <string: 0x7fff76257f40> { length = 22, contents = "Connection interrupted" }
}>